lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -5,45 +5,46 @@ defmodule ExUnit.Runner do
 
   @rand_algorithm :exs1024
 
-  def run(opts, load_us) do
-    {:ok, manager} = EM.start_link()
-    {:ok, stats} = EM.add_handler(manager, ExUnit.RunnerStats, opts)
-    {opts, config} = configure(manager, opts)
+  @type manager :: ExUnit.EventManager.manager()
+
+  def run(load_time, opts) when (is_integer(load_time) or is_nil(load_time)) and is_list(opts) do
+    opts = normalize_opts(opts)
 
+    {:ok, manager} = EM.start_link()
+    {:ok, stats_pid} = EM.add_handler(manager, ExUnit.RunnerStats, opts)
+    config = configure(opts, manager, self(), stats_pid)
     :erlang.system_flag(:backtrace_depth, Keyword.fetch!(opts, :stacktrace_depth))
 
-    {run_us, _} =
```

Comment:
I think we are losing important information here, which is that the runtime is in `us`.

Reply:
Agree with @josevalim. I would use `usec` though.

Reply:
So I would go with `runtime_us`.

Reply:
The reason I changed it its because `run_us` and `load_us` since it has a meaning in English, it was confusing, I have never seen that term before and neither it is used in Elixir core, only one time here.
Took me a while to figure it out.

If I had read `run_usec` I would have figured it out instantly